modules/compliance-apply-remediation-for-customized-mcp.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: PROCEDURE
+[id="complianc-operator-apply-remediation-for-customized-mcp"]
+= Applying remediation when using customized machine config pools
+
+When you create a custom `MachineConfigPool`, add a label to the `MachineConfigPool`.
```

Comment:
not my space, so feel free to ignore if this doesn't seem helpful, but skimming here I wasn't immediately clear on why you'd want to do this.  From [the bug][1], it became clear that it was setting up the pool to be found by the KubeletConfig's `machineConfigPoolSelector`.  Is it worth adding:

> ... so the KubeletConfig's `machineConfigPoolSelector` can match it.

or some such here, to make that context more discoverable?

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=2047742#c0

Reply:
Addressed